wikidatarefisland/data_model/__init__.py

```diff
@@ -1,3 +1,7 @@
+from .value_matchers import ValueMatchers, StringValue, QuantityValue
```

Comment:
Looking at this, while it's great we have them inside data_model but the `__all__ ` looks like it has low cohesion. I would say let's move either move schemaorg stuff to data_model.schemaorg or move this to data_model.wikibase (one level deeper) but it's not a big deal tbh.

Reply:
Yeah, it was bugging me too. Will move this one level deeper. Thanks :)